src/systemtest/java/com/ericsson/ei/systemtest/testexecutionflow/TestExecutionFlowSteps.java

```diff
@@ -0,0 +1,105 @@
+package com.ericsson.ei.systemtest.testexecutionflow;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.test.context.support.AbstractTestExecutionListener;
+
+import com.ericsson.ei.systemtest.utils.Config;
+import com.ericsson.ei.systemtest.utils.StepsUtils;
+import com.ericsson.eiffelcommons.utils.ResponseEntity;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+@Ignore
+public class TestExecutionFlowSteps extends AbstractTestExecutionListener{
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestExecutionFlowSteps.class);
+    private static final String JENKINS_TOKEN = "123";
+    private static final String JENKINS_JOB_XML = "jenkinsJobTemplate.xml";
```

Comment:
You should use the commons XML creator.
Implement and make use of JenkinsXmlData.class

Reply:
@e-edling-ericsson will take care of this